support/releaseinfo/fixtures/4.10-image-references.json

```diff
@@ -0,0 +1,2507 @@
+{
```

Comment:
how does this file help to test the new changes?
wouldn't we need to include a 4.10 bootimages.yaml file as we have for 4.8 and then modify `/test/main` `Extract` to account for it?

Reply:
I did include the file: `4.10-installer-coreos-bootimages.yaml` not sure if that's the one you are talking about.

Reply:
and I see a testcase here: https://github.com/openshift/hypershift/blob/6786c4ec968565c4ebdede2285ebda3343b1ee50/support/releaseinfo/registryclient/test/main.go#L72:L77

I will take a look at it.

Reply:
That test case is to test the performance of the image extraction code and not necessarily the format of the file.
The new format is getting tested here: https://github.com/openshift/hypershift/pull/690/files#diff-41538f4188cc1a87d29ff10f503cf3dc06cc6838128c9b4280d14514e0f9873dR19
However, I think we could benefit from some assertions that the new platform data is getting included in the parsed result.

Reply:
right, that's not validating at all that this new code works as expected, see also https://github.com/openshift/hypershift/blob/main/support/releaseinfo/registryclient/test/main.go#L72-L76

Reply:
Like I mentioned, that's for testing the performance of the image extraction code. I don't think we need to validate other versions there. All we're testing is whether the extracted data is the same as what was expected.

Reply:
> However, I think we could benefit from some assertions that the new platform data is getting included in the parsed result.

any specific assertion you are looking for?

Reply:
Just that you can read the data for the new platform

Reply:
@csrwng added the test in the latest commit, ptal